contents/handbook/people/team-structure/app-west/objectives.mdx

```diff
@@ -1,10 +1,18 @@
-* **Proposed Objective:**
-    * Increase the % of customers successfully using multiple core tools
-* **Proposed Key Results:**
-    * Launch all missing competitive session recording features like console log, network tab recording, playlist mode
-    * Improve feature flag capabilities (Pending specifics)
-    * Stretch: Proof of concept for mobile session recording
-* **Rationale:**
-    * This should drive us to focus on:
-        * Improving core features, such as session recording and feature flags to make them more valuable and attractive to use
-        * Integrating core features together to drive people to use multiple feature and make them even more successful on PostHog
\ No newline at end of file
+- Objectives: Enable customers to query the clickhouse database with SQL and return table form data (and maybe graphing) with 5 happy companies for the feature launch
+    - Key result:
+        - _Team to fill in_
+    - Why?
```

Comment:
Would like to know how to make a confident decision here on adding a query interface vs improving the existing capabilities